src/quo2/components/settings/accounts/view.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.settings.accounts.view
+  (:require [quo.react-native :as rn]
+            [quo2.components.avatars.account-avatar :as av]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.settings.accounts.style :as style]))
+
+(defn card-background [{:keys [custom-color]}]
+  [:<>
+   [rn/view {:style (style/background-top custom-color)}]
+   [rn/view {:style (style/background-bottom)}]])
+
+(defn avatar [{:keys [_color _icon] :as avatar-props}]
```

Comment:
I use those vars as some kind of code documentation.

If we only pass `avatar-props` it could be a little harder to know what keys this function needs to work.

That's the reason why I added them, should I remove them?

Reply:
It's an interesting approach @ulisesmac, but it can quickly get outdated due to the lack of usage of the bindings. In essence, the function `avatar` doesn't care about what's inside `avatar-props`, as long as it's a map, it just needs to assoc a default `:size`, so to me it's the opposite, not having the destructuring of the map helps me realize what's inside the map is not important.

Reply:
I think it's a matter of style and preferences, I understand your point of view. I removed the destructured parameters in ac8644a301daad136ded2f3290379f8b20b611ae.

Reply:
It really is, although I argued in favor of removing them, it's subjective.